flowy/src/solar.rs

```diff
@@ -156,14 +156,24 @@ impl Timetable {
         ret
     }
 
+    /// Returns the time of a solar event contained in the internal `Map`
+    /// - st: The SolarTime of interest
+    pub fn get(&self, st: &SolarTime) -> std::option::Option<&f64> {
+        self.timetable.get(st)
+    }
+
     /// Simple function to retrieve only sunset and sunrise times
     /// Returns a tuple (sunrise, sunset)
     pub fn get_sunrise_sunset(&self) -> (String, String) {
         // Index into the HashMap using SolarTime Enum
         let sunrise: String =
-            unix_to_normal_time(self.timetable.get(&SolarTime::Sunrise).unwrap().round() as i64);
+            unix_to_local(self.timetable.get(&SolarTime::Sunrise).unwrap().round() as i64)
```

Comment:
This function now has been deprecated?

Reply:
No, I just renamed it and delegated the String formatting to the caller. This way any function that needs time formatting can just use any format. I think I could make it simpler/less verbose to use making it receive the format string code as a parameter then formatting internally

Reply:
Where are we using this function? I don't see it in the code? If it is useless, let's deprecate it?

Reply:
Actually, I would suggest you just change the way we get sunrise and sunset times in flowy/lib.rs and keep using this function as it allows for a higher level of abstraction.

Reply:
The only place this function was used in was `generate_config_solar` but my implementation does not use it anymore. And if you're referring to lines 81-82, I already changed them to use seconds directly

Reply:
Yes I know that. I am asking you to change your implementatio so that ```generate_config_solar``` uses it  ```get_sunrise_sunset``` again

Reply:
Sure! Will do asap

Reply:
Great! Just refactor ```get_sunrise_sunset``` to ensure it works with you changes.